packages/php/src/ServiceProvider.php

```diff
@@ -14,7 +14,7 @@ public function boot()
         ], 'config');
 
         $this->publishes([
-            __DIR__ . '/handler.dist.php' => $path = app_path('Handler/ReadMe.php'),
```

Comment:
This wasn't a problem, but [Psalm](https://psalm.dev/) noticed it and since it's unnecessary I removed it.